digital_image_processing/test_digital_image_processing.py

```diff
@@ -96,9 +96,12 @@ def test_nearest_neighbour(
 
 
 def test_local_binary_pattern():
-    file_path = "digital_image_processing/image_data/lena.jpg"
+    from os import getenv  # Speed up our Continuous Integration tests
 
-    # Reading the image and converting it to grayscale.
+    file_name = "lena_small.jpg" if getenv("CI") else "lena.jpg"
+    file_path = f"digital_image_processing/image_data/{file_name}.jpg"
```

Comment:
```suggestion
    file_path = f"digital_image_processing/image_data/{file_name}"
```
The build is failing because your filepath is "lena.jpg.jpg"

Reply:
My mistake.  :-(